src/main/java/hudson/plugins/git/GitChangeLogParser.java

```diff
@@ -85,6 +87,8 @@ public List<GitChangeSet> parse(@NonNull List<String> changelog) {
         // Parse the log file into GitChangeSet items - each one is a commit
         try (Stream<String> lineStream = Files.lines(changelogFile.toPath(), StandardCharsets.UTF_8)) {
             return new GitChangeSetList(build, browser, parse(lineStream.iterator()));
+        } catch (InvalidPathException e) {
+            throw new IOException(e);
```

Comment:
It was a bit risky to use `changelogFile.toPath()` since this can throw `InvalidPathException` which is not assignable to `IOException`, and who knows if some caller is depending on this to always throw Here we rethrow `InvalidPathException` as `IOException`. Probably unnecessary, but better safe than sorry and I am being a bit paranoid since this is a widely installed plugin.